tsconfig-base.json

```diff
@@ -0,0 +1,26 @@
+{
+  "compilerOptions": {
+    "lib": ["WebWorker", "DOM", "DOM.Iterable"],
+    "target": "es2020",
+    "module": "es2020",
+    "moduleResolution": "node",
+    "allowJs": true,
+    "jsx": "react",
+    "strict": true,
+    "esModuleInterop": true,
+    "experimentalDecorators": true,
+    "emitDecoratorMetadata": true,
+    "noEmit": true,
+    "resolveJsonModule": true,
+    "skipLibCheck": true,
+    "types": ["node", "jest", "@testing-library/jest-dom"]
+  },
+  // since ts-node compiles ts on the fly and then uses node, it needs to
+  // compile the scripts to commonjs (or node will complain about the requires)
```

Comment:
I'm impressed nothing got angry about this, since JSON doesn't support comments 🤔

Reply:
Yeah, strictly speaking this isn't JSON.  Thankfully, the TS were kind and let us use this variant that allows comments.  For this reason, basically

Reply:
Don't tempt me I'll write a test for it 😛